README.md

```diff
@@ -1,131 +1,171 @@
 
-## Development
+- Name: Name of the badge.
+- Description: Description of the badge.
+- Image: Only emojis are allowed. You must input the emoji name as you would do to add it to a message (e.g. `:+1:` or `:smile:`). Custom emojis are also allowed.
+- Type: The type of the badge. This list will show only types you have permissions to create.
+- Multiple: Whether this badge can be granted more than once to the same person.
 
-To avoid having to manually install your plugin, build and deploy your plugin using one of the following options.
+### Details about Multiple
+All badges can be assigned to any number of persons. What the Multiple setting controls is whether this badge can be granted more than once to the same person. For example, a "Thank you" badge should be granteable many times (many people can be thankful to you in more than one ocassion). This badge should have the Multiple checkbox marked. But a "First year in the company" badge should be granted just once (it makes no sense to have 2 first year in the company). That badge should have the Multiple checkbox unmarked.
 
-### Deploying with Local Mode
+### Granting a badge
+In order to grant a badge, you must have permissions to grant it.
+There are two ways to open the grant dialog. One of them is using the `/badges grant` command. Another one is clicking on the "Grant badge" link available in the Profile Popover visible when you click on someones username.
 
-If your Mattermost server is running locally, you can enable [local mode](https://docs.mattermost.com/administration/mmctl-cli-tool.html#local-mode) to streamline deploying your plugin. Edit your server configuration as follows:
+![Screenshot from 2022-03-16 11-47-14](https://user-images.githubusercontent.com/1933730/158573673-723e77a2-6d58-4aa5-8a89-6adcbce50e13.png)
 
-```json
-{
-    "ServiceSettings": {
-        ...
-        "EnableLocalMode": true,
-        "LocalModeSocketLocation": "/var/tmp/mattermost_local.socket"
-    }
-}
-```
+The dialog looks like this:
 
-and then deploy your plugin:
-```
-make deploy
-```
+![Screenshot from 2022-03-16 11-51-05](https://user-images.githubusercontent.com/1933730/158573834-70ea72b0-4a03-4b09-a694-751c0ca1ba04.png)
 
-You may also customize the Unix socket path:
-```
-export MM_LOCALSOCKETPATH=/var/tmp/alternate_local.socket
-make deploy
-```
+- User: The user you want to grant the badge to (may be prepopulated if you clicked the grant button from the profile popover, or added the username in the command).
+- Badge: The badge you want to grant (may be prepopulated if you added the badge id in the command).
+- Reason: An optional reason why you are awarding this badge. (Specially useful for badges like "Thank you")
+- Notify on this channel: If you mark this, a message from the badges bot will be posted in the current channel, letting all know that you granted this badge to that person.
+When a badge is granted, three things may happen:
```

Comment:
Given that the first bullet point below WILL always happen, consider rewording the introductory sentence as: 
"When a badge is granted, the user who received the badge will receive a DM from the badges bot letting them know that they have been awarded a badge."

Then, you can introduce the 2 potential outcomes as: "In addition, the following may happen:"